staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/generic/webhook.go

```diff
@@ -140,7 +140,10 @@ func (a *Webhook) ValidateInitialization() error {
 
 // ShouldCallHook returns invocation details if the webhook should be called, nil if the webhook should not be called,
 // or an error if an error was encountered during evaluation.
-func (a *Webhook) ShouldCallHook(h webhook.WebhookAccessor, attr admission.Attributes, o admission.ObjectInterfaces) (*WebhookInvocation, *apierrors.StatusError) {
+func (a *Webhook) ShouldCallHook(h webhook.WebhookAccessor, attr admission.Attributes, o admission.ObjectInterfaces, clusterName string) (*WebhookInvocation, *apierrors.StatusError) {
+	if h.GetCluster() != clusterName {
+		return nil, nil
+	}
```

Comment:
this looks fine for CRDs, but probably not enough for APIBindings. Before merging this PR I would like to see a kcp "proof" PR showing in e2e that admission works for APIBindings.

Reply:
I discussed with Shawn yesterday. I think we might want to split this into

1. Just getting webhooks working within a single workspace
2. Support for webhooks for APIBindings, after we implement identity

WDYT?

Reply:
I discussed with Shawn yesterday. I think we might want to split this into

1. Just getting webhooks working within a single workspace
2. Support for webhooks for APIBindings, after we implement identity

WDYT?